src/navigator_data_ingest/main.py

```diff
@@ -126,7 +127,10 @@ def main(
 
     _LOGGER.info(f"Loading Law/Policy document data from '{input_file_path}'")
 
-    document_generator = LawPolicyGenerator(input_file_path)
+    json_data = read_s3_json_file(input_file_path)
```

Comment:
Should we validate the data type here with something like: 

read_s3_json_file() -> input_data

where input_data might be something like: 

dict[Literal['new_documents]: dict, [Literal['updated_documents']: dict] 

Then we access via:

json_data.new_documents so we assert that we won't get a key error?

Reply:
Yeah, this is a nice idea, but I'd probably suggest using a pydantic model that specifies the keys... :smile: 
It'll probably going to look like the below for a while (when we have the new db code finished): ```python
class InputData(BaseModel):
    new_documents: Sequence[DocumentParserInput]
    updated_documents: Sequence[DocumentParserInput]```